packages/remix-server-runtime/server.ts

```diff
@@ -1019,6 +1043,39 @@ async function assertResponsesMatch(_a: Response, _b: Response) {
   }
 }
 
+function stripStackTraceFromDefaultErrorBoundary(text: string) {
+  let openPreStart = text.indexOf("<pre ");
+  let openPreEnd = text.indexOf('">', openPreStart + 1);
+  let closePre = text.indexOf("</pre>", openPreEnd + 1);
+  let error = text.substring(openPreEnd + 2, closePre);
+  let stackStart = error.indexOf("\n");
+  let errorWithoutStack = error.substring(0, stackStart);
+  return (
+    text.replace(error, "ERROR REMOVED") + "\n\nError:\n" + errorWithoutStack
```

Comment:
Here we end up asserting the full HTML without `<script>window.__remixContext = {...}</script>` concatenated with the stringifed `remixContext` with `stack` replaced